CHANGELOG.md

```diff
@@ -5,6 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 3.2.0
+
+### Added
+* sink: implement mTLS-enabled http client by @roobre in https://github.com/newrelic/nri-kubernetes/pull/338
+* Feature/restart count delta by @sigilioso in https://github.com/newrelic/nri-kubernetes/pull/382
+  a New metric has been added. It reports the number of restart since last sample: `restartCountDelta`
+* Dockerfile: simplify binary path, ensure 755 permissions by @roobre in https://github.com/newrelic/nri-kubernetes/pull/364
+
+### Fixed
+* fix(pendingPods): some metrics are missing from KSM by @paologallinaharbur in https://github.com/newrelic/nri-kubernetes/pull/404
+  The metric `isReady` is now reported for pending pod as well
```

Comment:
```suggestion
* `isReady` metric is now correctly reported as `false` (rather than `NULL`) for pending pods by @paologallinaharbur in https://github.com/newrelic/nri-kubernetes/pull/404
```